frontend/apps/app/components/SessionsNewPage/SessionsNewPage.tsx

```diff
@@ -2,17 +2,28 @@
 
 import { ArrowRight, Button } from '@liam-hq/ui'
 import type { ChangeEvent, FC } from 'react'
-import { useActionState, useEffect, useRef } from 'react'
+import { useActionState, useEffect, useRef, useTransition } from 'react'
+import type { Projects } from '../CommonLayout/AppBar/ProjectsDropdownMenu/services/getProjects'
 import styles from './SessionsNewPage.module.css'
 import { createSession } from './actions/createSession'
+import { getBranches } from './actions/getBranches'
 
-export const SessionsNewPage: FC = () => {
-  const textareaRef = useRef<HTMLTextAreaElement>(null)
+type Props = {
+  projects: Projects | null
+}
 
+export const SessionsNewPage: FC<Props> = ({ projects }) => {
+  const textareaRef = useRef<HTMLTextAreaElement>(null)
+  const [_isPending, startTransition] = useTransition()
```

Comment:
Since _isPending is unused, it might be better to write [, startTransition] as a tuple destructuring expression.
```suggestion
  const [, startTransition] = useTransition()
```